source/frontend/src/components/common/mapFSM/useMapSearch.tsx

```diff
@@ -28,10 +28,48 @@ export const useMapSearch = () => {
   const loadPimsProperties = pimsPropertyLayerService.loadPropertyLayer.execute;
   const fullyAttributedServiceFindByPin = fullyAttributedService.findByPin;
   const fullyAttributedServiceFindByPid = fullyAttributedService.findByPid;
+  const fullyAttributedServiceFindByPLanNumber = fullyAttributedService.findByPlanNumber;
```

Comment:
within useFullyAttributedParcelMapLayer.tsx, pass the forceExactMatch param as true to resolve this issue.